accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -1008,6 +1011,33 @@ void main() {
 }
 ```
 
+### Constant assertions
+
+Currently assertions are enabled only when the compiler is passed an `enable-asserts` flag.
+With null safety, when an assert is evaluated as part of a constant invocation of a `const`
```

Comment:
We could call them `an assertion` as a rule, and only use `assert` when referring to actual code. The language specification has `assert statement` which is a statement that is an assertion, but otherwise it refers to assertions as `assertion`s. Might as well do that here, too.

Reply:
Good idea. I think I did that deliberately when I introduced assertions into initializer lists (where they are not statements).